### PR TITLE
print relative path in summary output

### PR DESCRIPTION
**Pull request by charlierudolph**
@jbpros @samccone 

Absolute path isn't very helpful.

### CODE REVIEW

**Comment by jbpros** [comment]
Thanks for all the stuff you submitted @charlierudolph. I'm currently on vacation, will be back in a few days.

File: spec/cucumber/listener/summary_formatter_spec.js [hunk 1]
```diff
@@ -274,13 +274,14 @@ describe("Cucumber.Listener.SummaryFormatter", function () {
...
 
   describe("storeFailedScenario()", function () {
-    var failedScenario, name, uri, line, string;
+    var failedScenario, name, relativeUri, uri, line, string;
 
     beforeEach(function () {
       name           = "some failed scenario";
-      uri            = "/path/to/some.feature";
+      relativeUri    = "path/to/some.feature";
+      uri            = process.cwd() + "/" + relativeUri;
```
Comment (samccone) [inline spec/cucumber/listener/summary_formatter_spec.js]: lets use path.resolve here :)

Comment (samccone) [inline spec/cucumber/listener/summary_formatter_spec.js]: oh nm you want relative to the cwd... ok, nevertheless lets use path.join :)

**Comment by jbpros** [comment]
Merged in as 33afb4dde4bfa71b939b3a7ccc80aa05ddd71378

**Comment by jbpros** [comment]
Mmm actually I found a bug. If you run cucumber from within a nested directory in the features folder, the reported paths will be wrong (`feature:12` instead of the relative file path like `../my.feature:12`).

I'm reverting the commit and reopening. @charlierudolph can you fix it please?

**Comment by jbpros** [comment]
Reverted merge in b09f7f8

**Comment by charlierudolph** [comment]
@jbpros updated

**Comment by jbpros** [comment]
Thanks mate, it works like a charm. Merged in as 7571aa639125b9aed622ef0d36d4493c4fbb9137

**Pull request closed without merging**